Sources/News.php

```diff
@@ -225,11 +225,11 @@ function ShowXmlFeed()
 	// Allow mods to add extra namespaces and tags to the feed/channel
 	$namespaces = array(
 		'rss' => array(),
-		'rss2' => array('atom' => 'http://www.w3.org/2005/Atom'),
-		'atom' => array('' => 'http://www.w3.org/2005/Atom'),
+		'rss2' => array('atom' => 'https://www.w3.org/2005/Atom'),
+		'atom' => array('' => 'https://www.w3.org/2005/Atom'),
```

Comment:
The feeds won't validate with anything but the canonical http URL. Try one in the [W3 feed validator](https://validator.w3.org/feed/). Using https in the namespace declaration URL results in the following error message:

```
Sorry

This feed does not validate.
line 2, column 0: feed is in an invalid namespace: https://www.w3.org/2005/Atom [help]
<feed xmlns="https://www.w3.org/2005/Atom">

In addition, interoperability with the widest range of feed readers could be improved by implementing the following recommendation.
line 2, column 0: Use of unknown namespace: https://www.w3.org/2005/Atom [help]
<feed xmlns="https://www.w3.org/2005/Atom">
```